.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+.idea
```

Comment:
whats the .idea file? if its not being used then just delete it

Reply:
Not sure, I didn't create it, so was concerned it was a system file, if its not important I can delete it.

Reply:
Seems to be a php storm file: 
https://rider-support.jetbrains.com/hc/en-us/articles/207097529-What-is-the-idea-folder-